samples/subsys/mgmt/mcumgr/smp_svr/overlay-udp-ipv6.conf

```diff
@@ -0,0 +1,15 @@
+# Enable the UDP mcumgr transports.
+CONFIG_MCUMGR_SMP_UDP=y
+CONFIG_MCUMGR_SMP_UDP_IPV6=y
+
+# Network settings
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_CONNECTION_MANAGER=y
+CONFIG_NET_BUF_USER_DATA_SIZE=24
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_ETH_MCUX_0_UNIQUE_MAC=y
```

Comment:
This settings seems to be the only one that is specific for the k64f board. Is it required to run on this board? If not, can you remove it here and in the ipv4 overlay file?